packages/prices/utils/price.ts

```diff
@@ -90,7 +90,7 @@ export const getCurrencyFromPriceResponse = (
 		currency_suffix: suffix,
 	} = currencyData;
 
-	return {
+	return <Currency>{
```

Comment:
Casting is not required here, it will validate returned object against the return type provided in function definition at line 78

Reply:
Oh okay, the IDE was complaining about it 🤔